modules/serverless-kafka-source-odc.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * serverless/event_workflows/serverless-pingsource.adoc
+
+[id="serverless-kafka-source-odc_{context}"]
+= Creating a KafkaSource using the Developer Perspective
+
+You can create and verify a Kafka event source from the {product-title} web console.
+
+.Prerequisites
+To create a Kafka event source using the *Developer* perspective, ensure that:
+
+* The {ServerlessOperatorName}, Knative Eventing, and the `KnativeKafka` custom resource are installed on your {product-title} cluster.
+* You have logged in to the web console.
+* You are in the *Developer* perspective.
+* You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {product-title}.
+
+.Procedure
+
+. Navigate to the **Add** page and select **Event Source**.
+. In the **Event Sources** page, select **Kafka Source** in the **Type** section.
+. Configure the **Kafka Source** settings:
+.. Add a comma-separated list of **Bootstrap Servers**.
+.. Add a comma-separated list of **Topics**.
+.. Add a **Consumer Group**.
```

Comment:
optional

Reply:
None of these are optional in ODC - should they be @matzew ?